addons/sourcemod/scripting/surftimer/sql.sp

```diff
@@ -8675,6 +8691,80 @@ public int ChooseMapMenuHandler(Menu menu, MenuAction action, int param1, int pa
 		delete menu;
 }
 
+public void db_selectMapCurrentWR(Handle owner, Handle hndl, const char[] error, any pack)
+{
+	if (hndl == null)
+	{
+		LogError("[SurfTimer] SQL Error (db_selectMapCurrentWR): %s", error);
+		return;
+	}
+
+	if (SQL_HasResultSet(hndl) && SQL_FetchRow(hndl))
+	{
+		ResetPack(pack);
+		char szSteamId[32], szStyleType[5];
+		ReadPackString(pack, szSteamId, sizeof(szSteamId));
+		ReadPackString(pack, szStyleType, sizeof(szStyleType));
+		int iLevel = ReadPackCell(pack);
+		CloseHandle(pack);
```

Comment:
I prefer to close the handle at the end of this callback, outside of `if (SQL_HasResultSet(hndl) && SQL_FetchRow(hndl))` and in `if (hndl == null)`. Otherwise it could leak.

Reply:
I already saw this a few times and will update this in a further pull request.